src/Neovim.ts

```diff
@@ -61,13 +61,20 @@ export default class Neovim {
 
   openFile = async (file: TFile | null) => {
     if (!file) return;
-    if (!this.settings.supportedFileTypes.includes(file.extension)) return;
+    if (
+      !this.settings.supportedFileTypes.includes(file.extension) ||
+      // condition for the excalidraw filetype
+      (file.extension == "md" &&
+        !this.settings.supportedFileTypes.includes("excalidraw") &&
+        file.name.includes(".excalidraw"))
```

Comment:
In this case it might be better you use `file.name.endsWith(".excalidraw")`; since there could be files created by the user of other plugins that have `.excalidraw` somewhere else in the file name and we wouldn't want to exclude them by accident :slightly_smiling_face:

Reply:
That's way safer!! thank you! 🙂